src/ui.container-modal-confirm/index.stories.js

```diff
@@ -58,7 +58,9 @@ const EnumVariantTemplate = (args, { argTypes } = {}) => ({
   components: { UModalConfirm, UButton, URow },
   setup() {
     function onClick(value) {
-      argTypes[args.enum].name === "color" ? (args.color = value) : (args.width = value);
+      argTypes[args.enum].name === "confirmColor"
```

Comment:
why `confirmColor`?

Reply:
Because `color` prop has been changed to `confirmColor`. If I don`t change its name here - I get an error and story is either not rendered at all, or is rendered without colors.